<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QCheckBox" name="check_Auto_Resize">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>391</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Resize main windows automatically to fit the opened videos</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_File_Associations">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>210</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Set File &amp;Associations</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_File_Associations">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>210</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>Set to the default program for opening supported video files.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
       </widget>
       <widget class="QCheckBox" name="check_Playback_Loop">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>401</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Play video in a loop</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_Distortion_Measures">
       <attribute name="title">
        <string>Distortion Measures</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Available Measures</string>
        </property>
       </widget>
       <widget class="QListWidget" name="measureList">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>91</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="measureOptions" native="true">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>311</width>
          <height>221</height>
         </rect>
        </property>
        <widget class="QCheckBox" name="check_Show_Distortion_Map">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>291</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Enable Distortion Map</string>
         </property>
        </widget>
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>301</width>
           <height>115</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="layout_Measure_Option">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Upper Limit</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spin_Overshoot">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Lower Limit</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spin_Undershoot">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Overshoot Color</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="button_Overshoot">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Undershoot Color</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="button_Undershoot">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_Measure_Info">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>301</width>
           <height>61</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Show Information about the measure here</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Options</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
